index.js

```diff
@@ -67,6 +68,65 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener("click", function(e){
+    // console.log(e.target)
+    toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
-/***** Deliverable 3 *****/
+const buttonLikes = document.querySelector("#profile button.like-button")
+let increaseLike = traveler.likes
+buttonLikes.addEventListener("click", function(e) {
```

Comment:
Good, but what if there were multiple travelers who all had likes and I needed to find only the one that I clicked on? Here's one way we could approach that..

```javascript
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
With ES6 notation
```javascript
const likeBtn = document.querySelector(".like-button");
const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
For more info on previousElementSibling, check here